H2TauTau/python/proto/samples/fall15/higgs_susy.py

```diff
@@ -4,26 +4,27 @@
+
+HiggsSUSYGG80   = creator.makeMCComponent("HiggsSUSYGG80"  , "/SUSYGluGluToHToTauTau_M-80_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM"  , "CMS", ".*root", 1.0)
+HiggsSUSYGG90   = creator.makeMCComponent("HiggsSUSYGG90"  , "/SUSYGluGluToHToTauTau_M-90_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM"  , "CMS", ".*root", 1.0)
+HiggsSUSYGG100  = creator.makeMCComponent("HiggsSUSYGG100" , "/SUSYGluGluToHToTauTau_M-100_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG110  = creator.makeMCComponent("HiggsSUSYGG110" , "/SUSYGluGluToHToTauTau_M-110_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG120  = creator.makeMCComponent("HiggsSUSYGG120" , "/SUSYGluGluToHToTauTau_M-120_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG130  = creator.makeMCComponent("HiggsSUSYGG130" , "/SUSYGluGluToHToTauTau_M-130_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG140  = creator.makeMCComponent("HiggsSUSYGG140" , "/SUSYGluGluToHToTauTau_M-140_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG160  = creator.makeMCComponent("HiggsSUSYGG160" , "/SUSYGluGluToHToTauTau_M-160_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG180  = creator.makeMCComponent("HiggsSUSYGG180" , "/SUSYGluGluToHToTauTau_M-180_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG200  = creator.makeMCComponent("HiggsSUSYGG200" , "/SUSYGluGluToHToTauTau_M-200_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG250  = creator.makeMCComponent("HiggsSUSYGG250" , "/SUSYGluGluToHToTauTau_M-250_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v2/MINIAODSIM" , "CMS", ".*root", 1.0)
+HiggsSUSYGG300  = creator.makeMCComponent("HiggsSUSYGG300" , "/SUSYGluGluToHToTauTau_M-300_TuneCUETP8M1_13TeV-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v2/MINIAODSIM" , "CMS", ".*root", 1.0)
+# HiggsSUSYGG350  = creator.makeMCComponent("HiggsSUSYGG350" , "/SUSYGluGluToHToTauTau_M-350_TuneCUETP8M1_13TeV-pythia8/RunIISpring15MiniAODv2-74X_mcRun2_asymptotic_v2-v1/MINIAODSIM"                 , "CMS", ".*root", 1.0)
```

Comment:
Not sure I understand why you comment out this samples since it should be there
https://cmsweb.cern.ch/das/request?view=list&instance=prod%2Fglobal&input=site+dataset%3D%2FSUSYGluGluToHToTauTau_M-350_TuneCUETP8M1_13TeV-pythia8%2FRunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1%2FMINIAODSIM